pygeoapi/util.py

```diff
@@ -359,6 +403,22 @@ def get_provider_default(providers):
     return default
 
 
+class JobStatus(Enum):
+    """
+    Enum for the job status options specified in the WPS 2.0 specification
+    """
+
+    #  From the specification
+    accepted = 'accepted'
+    running = 'running'
+    successful = 'successful'
+    failed = 'failed'
+    dismissed = 'dismissed'
+
+    #  Alternative namings used in existing codebase
+    finished = successful  # TODO should this status be used?
```

Comment:
Could be removed if use of `"finished"` is purged

Reply:
which may already be the case, it's possible this comment is out of date now

Reply:
Referring only to `finished`, not `dismissed`

Reply:
Updated to align to https://github.com/opengeospatial/ogcapi-processes/blob/master/core/openapi/schemas/statusInfo.yaml#L8